src/Bindings/ManualBindings_World.cpp

```diff
@@ -110,6 +110,73 @@ static int tolua_cWorld_ChunkStay(lua_State * tolua_S)
 
 
 
+static int tolua_cWorld_ForEachLoadedChunk(lua_State * tolua_S)
+{
+	// Exported manually, because i don't know what i'm doing...
```

Comment:
Wanted a comment from someone who knows how it works: can it be exported automatically? Does it need to be exported automatically? Is manual export okay?